text-minimessage/src/main/java/net/kyori/adventure/text/minimessage/tag/standard/StandardTags.java

```diff
@@ -59,7 +59,9 @@ private StandardTags() {
         RainbowTag.RESOLVER,
         ResetTag.RESOLVER,
         NewlineTag.RESOLVER,
-        TransitionTag.RESOLVER
+        TransitionTag.RESOLVER,
+        ProgressTag.RESOLVER,
+        RepeatTag.RESOLVER
```

Comment:
Very nitpicky, but does checkstyle allow us to do trailing commas here?

Reply:
That's how it was done before too.